opennms-container/core/plugins.sh

```diff
@@ -9,29 +12,28 @@ export DEPLOY_FOLDER="/opt/usr-plugins"
 
 mkdir $DEPLOY_FOLDER 
 
-apt-get update
-apt-get install -y python3-pip wget curl jq
+microdnf -y install cpio python3-pip jq
 pip3 install --upgrade cloudsmith-cli 
 
 mkdir ~/test
 cd ~/test || exit
-artifact_urls=$(cloudsmith list packages --query="opennms-alec-plugin version:$ALEC_VERSION format:deb" opennms/common -F json  | jq -r '.data[].cdn_url')
+artifact_urls=$(cloudsmith list packages --query="opennms-alec-plugin version:$ALEC_VERSION format:rpm" opennms/common -F json  | jq -r '.data[].cdn_url')
 for url in $artifact_urls; do 
- wget "$url"
+ curl -L -O "$url"
```

Comment:
```suggestion
 curl -sSF -L -O "$url"
```

With curl, `-sSf` everywhere. :)